components/package-template/src/sbin/start-clp.sh

```
@@ -10,8 +10,81 @@ common_env_path="$script_dir/.common-env.sh"
+        echo "For details, see: https://docs.yscope.com/clp/main/user-guide/telemetry"
+        echo ""
+        echo "You can disable telemetry at any time by setting CLP_DISABLE_TELEMETRY=true"
+        echo "or by blocking https://telemetry.yscope.io at the network level."
+        echo ""
+        read -r -p "Enable anonymous telemetry to help improve CLP? [Y/n] " telemetry_response
+        echo "================================================================================"
+
+        if [[ "$telemetry_response" =~ ^[Nn]$ ]]; then
+            # User opted out — persist to config
+            if [[ -f "$clp_config_path" ]]; then
+                echo "" >> "$clp_config_path"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)